client/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Form from "./components/Form";
```

Comment:
I've always been told best practice is to use uppercase C in /Components/ to keep with the naming system.